src/features/user/UserProfile/components/UserInfo.tsx

```diff
@@ -13,15 +13,23 @@ export default function UserInfo({
   handleAddTargetModalOpen,
 }: any) {
   const { t, ready } = useTranslation(['donate']);
+  console.log(userprofile,"user")
   return (
     <div className={styles.landingContent}>
+      {userprofile.type == "tpo" ? 
       <TreeCounter
+      handleAddTargetModalOpen={handleAddTargetModalOpen}
+      authenticatedType={authenticatedType}
+      target={userprofile.score.target}
+      planted={userprofile.score.personal}
+    /> : 
+    <TreeCounter
         handleAddTargetModalOpen={handleAddTargetModalOpen}
         authenticatedType={authenticatedType}
         target={userprofile.score.target}
         planted={userprofile.score.personal + userprofile.score.received}
```

Comment:
Why didn't you add the if clause `{userprofile.type == "tpo" ? userprofile.score.personal : userprofile.score.personal + userprofile.score.received}` here if only this line changed? (I have not yet tested this.)

Reply:
Done